iptables/service.sls

```diff
@@ -9,6 +9,31 @@ iptables_packages:
   pkg.installed:
   - names: {{ service.pkgs }}
 
+{%- if 'iptables-restore' in service.providers %}
+/usr/share/netfilter-persistent/plugins.d/15-ip4tables:
```

Comment:
1. you should not touch anything else than /tmp, /etc, /usr/local or /var
2. you should use scripts that are shipped with system package, I don't see a reason why to deliver them from here, common practice is to use distribution-specific ways when available